pages/login.php

```diff
@@ -113,18 +118,22 @@
 
 			<div class="form-elements">
           <img src="../images/red-avatar.png" alt="Avatar" class="avatar">
-          <?php 
-            if(isset($_GET["error"])) {
-              if ($_GET["error"] == "emptyInputLogin") {
+          <?php
+
+            if ( isset( $_GET["error"] ) ) {
```

Comment:
did you apply the `phpfmt` format? That is not required, just writing a readable code will do

Reply:
I think its because of format on save. Ok no problem i will remove the phpfmt settings.

Reply:
Ahh..okay